### PR TITLE
metrics: Enable new CI metrics baremetal.

### LINKED ISSUE #5401: enable metrics CI on new baremetal

### PR DESCRIPTION
**Pull request by GabyCT**
This PR enables a new CI metrics baremetal.

Fixes #5401

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/setup_env_ubuntu.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ source "/etc/os-release" || source "/usr/lib/os-release"
 source "${cidir}/lib.sh"
 
 echo "Update apt repositories"
```
Comment (dborquez) [inline .ci/setup_env_ubuntu.sh:14]: Thanks @GabyCT,  could you please comment this line to be consistent with the next commented line.

**Review by dborquez — CHANGES_REQUESTED** [review]
thanks @GabyCT LGTM, just resolve the suggestion made

**Comment by GabyCT** [comment]
@dborquez changes applied, thanks

**Review by dborquez — APPROVED** [review]

**Review by chavafg — APPROVED** [review]

**Pull request merged**